examples/golang/README.md

```diff
@@ -0,0 +1,41 @@
+# AWS Golang Client Example
+
+## Introduction
+
+This directory contains examples on how to use [AWS SDK for Go V1](https://github.com/aws/aws-sdk-go) to experience the RadosGW extensions with the S3 API.
```

Comment:
these are not extensions. this is the standard S3 functionality

Reply:
Thanks very much for the comments! I will modify the introduction in the README file.